create_jenkins_job.py

```diff
@@ -124,10 +125,18 @@ def main(argv=None):
         # configure manual triggered job
         job_name = 'ci_' + os_name
         job_data['cmake_build_type'] = 'None'
+        # For the aarch64 job:
+        #  * do it nightly (because it takes a very long time to run); and
+        #  * don't email on test failures (because there are many, likely
+        #    related to qemu)
+        if os_name == 'linux-aarch64':
+            job_data['time_trigger_spec'] = '0 11 * * *'
```

Comment:
In 3ffdb1f I reverted the changes to `ci_linux-aarch64` and create a new nightly job. I decided for it to be a `Debug` build (the exact build type doesn't matter that much; the goal here is to check for successful compilation):

http://ci.ros2.org/view/nightly/job/nightly_linux-aarch64_debug/